v2/tools/generator/internal/jsonast/swagger_type_extractor.go

```diff
@@ -328,6 +337,11 @@ func (extractor *SwaggerTypeExtractor) findARMResourceSchema(op spec.PathItem, r
 		}
 	}
 
+	if isResource == false && op.Head != nil {
+		// assume this is a resource?
```

Comment:
Do you know the answer to this yet?

Reply:
yeah I think this is OK, at least for now. If this is problematic it's because it may over-classify things

I've updated this comment to say:
> 		// assume this is a resource. It's possible this is too permissive and classifies some things as resources
>		// when they really aren't, but that's OK because anyway we only generate the resources we name in the configuration
>		// and classifying something as a resource here just makes it a candidate for mention in the config.